test/spec.smoke.js

```diff
@@ -0,0 +1,61 @@
+const testlab = require('@loopback/testlab');
+const createApp = require('../dist/application').createApp;
+const expect = testlab.expect;
+const validateApiSpec = testlab.validateApiSpec;
+
+const http = require('http');
+const Dredd = require('dredd');
+const port = 3030;
```

Comment:
Please use ephemeral port `0`, otherwise this test will fail when there is already some other process listening on port 3000, as may be the case in CI environments running multiple jobs in parallel on the same machine.